src/js/bootstrap-slider.js

```diff
@@ -1261,6 +1269,16 @@
 
 				return true;
 			},
+			_touchstart: function(ev) {
+				if (ev.changedTouches === undefined) {
+					this._trigger('slideStart');
+					return;
+				}
+
+				var touch = ev.changedTouches[0];
+				this.touchX = touch.pageX;
+				this.touchY = touch.pageY;
```

Comment:
Good call! It does and its actually the ultimate option as a fallback. This portion of the code won't work but that is ok, because the touchmove threshold check won't work either so this scenario everything will fallback to what it was before, which was functional at least.